### DISCUSSION TITLE
Configuration selection

### DISCUSSION THREAD

**Discussion by xNiux** [github0null/eide#249] [Ideas]
Hi !

Very good extension ! (Use it on Renesas Cortex MCU with FSP).
Just a possible usefull improvement : Add a Configuration selection button like "IAR Build" extension for example : 
![image](https://github.com/github0null/eide/assets/25155398/5b987bcb-09ed-41f2-97a7-a020733ea4ee)

To easily switch and have a permanent view on active configuration.

**Comment by github0null** [comment]
This feature looks good, I'll add it later !
